### PR TITLE
feat: blog subsection

### LINKED ISSUE #6474: Blog followup
Few things left after merging 

# Landing page
### articles
image
- currently it does not have subarticles, but two main one 
- there should be only one hero article at a time
- other articles should be in sections -  if we don't have enough (at least 3) - we can name it somewhat generally and put it there
- the structure is 1 main, 2, 3, 3...

### Title 
- the title should have gradient underneath 
image

### Hero card 
- please lower the line height of the article name. Something like 36px should work 
- the whole card could have a bigger height 
- cut the subheading at around 4 rows 
- make the card fixed, so there is always only one size for this specific screen size 
image
- gradient overflows the card 
image

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Needs check

- @kodadot/qa-guild please review
- @exezbcz please review

## Context

- [x] Closes #6474
- [x] smaller title size under featured article. because it wont fit if the title is too long
- [x] for now I set it as latest posts. you can visit [`/blog/?edge=true`](https://deploy-preview-6525--koda-canary.netlify.app/blog/?edge=true) to see post with 2 columns. for now I hide it, because it feels redundant
- [x] ensure to add `featured: true` for featured article

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)

## Screenshot 📸

<details><summary>screenshot</summary>
<p>

image

</p>
</details>

### CODE REVIEW

**Comment by exezbcz** [comment]
image
the margin between the hero article and subarticles is odd 
- should be 80px 

The hover effect should be a bit different. 
- should work the same as an article on the landing page 
image
- change of bg and stroke, cursor pointer - that is oki. 

here is the gradient for blog page 
bloglandinggradient


thank you!

**Comment by prury** [comment]
- mobile seems fine
- i would add more spacing between the featured one and the 2 column section (I'm using ?edge=true): 

image

once exez requests got added I'll check again, blog structure looking nice tho!

**Comment by yangwao** [comment]
> SonarCloud Quality Gate failed

welcome

File: pages/blog/index.vue [hunk 1]
```diff
@@ -7,40 +7,20 @@
...
+        class="content-list-cover"
+        :style="{ backgroundImage: `url(${post.attributes.image})` }"></div>
 
       <div class="content-list-card">
         <div>
           <div class="card-tag">• {{ post.attributes.tags }}</div>
-          <div class="title is-3">{{ post.attributes.title }}</div>
+          <p class="title is-4">
+            <a :href="getPermalink(post)">{{ post.attributes.title }}</a>
+          </p>
```
Comment (roiLeo) [inline pages/blog/index.vue:23]: I would use nuxt-link
```ts
<nuxt-link :to="getPermalink(post)">
    <div class="title is-4">{{ post.attributes.title }}</div>
</nuxt-link>
```

File: pages/blog/index.vue [hunk 2]
```diff
@@ -56,6 +36,48 @@
         </div>
       </div>
     </div>
+
+    <!-- article section with 2 grid cols -->
+    <div v-if="redesign">
+      <h2 class="title is-2">Tokens</h2>
+      <div class="content-list-grid content-list-grid-2">
+        <a
```
Comment (roiLeo) [inline pages/blog/index.vue:44]: same here use nuxt-link

File: pages/blog/index.vue [hunk 3]
```diff
@@ -56,6 +36,48 @@
...
+            </div>
+          </div>
+        </a>
+      </div>
+    </div>
+
+    <!-- article section -->
+    <h2 class="title is-2">Latest Posts</h2>
+    <div class="content-list-grid">
+      <a
```
Comment (roiLeo) [inline pages/blog/index.vue:65]: nuxt-link

**Review by roiLeo — APPROVED** [review]
small stuff otherwise lgtm

File: pages/blog/index.vue (hunk 1 above)
Comment (preschian) [reply pages/blog/index.vue]: updated, thank you

**Comment by preschian** [comment]
we have a bunch of articles now, so I just show the `tokens` section without `?edge=true`

**Comment by exezbcz** [comment]
Nice work! Starting to look good 

> * should work the same as an article on the landing page
>   image

- one issue I see on the cards is that the bottom stroke on the cover img changes as well, it should stay default and only the outer stroke on the whole container should change. 
- would it be possible to use the dynamic grid, like on Explore, on these screen sizes? 
image

**Review by vikiival — COMMENTED** [review]
~~Few~~

**Review by roiLeo — APPROVED** [review]
remove `edge` stuff if it's all good?

**Comment by vikiival** [comment]
Let's make one more recheck cc @exezbcz and let it roll

**Comment by exezbcz** [comment]
works for me, thanks!

**Pull request merged**